tests/drift/test_drift.py

```diff
@@ -841,3 +857,76 @@ def test_result_comparison_to_cbpe_plots_raise_no_exceptions(sample_drift_data):
         _ = result.compare(result2).plot()
     except Exception as exc:
         pytest.fail(f"an unexpected exception occurred: {exc}")
+
+
+def test_oo_behavior_univ_calc():  # noqa: D103
```

Comment:
Name could be a bit more descriptive, as this is a weird thing to test.

Maybe something like test_input_dataframes_are_not_altered_by_calculator